### DISCUSSION TITLE
Unable to finish the application for student developer pack

### DISCUSSION THREAD

**Discussion by sw3869** [community/community#113095] [GitHub Education]
### Hi everyone,

Hi, I encountered some difficulties while applying for student developer pack. I uploaded my student id and it says I have to filled in the billing information and 2FA to finish the academic affiliation application, so I did both of them. However, when I signed out and resign in to apply for the student developer pack again by submitting my student id, it says "Your application cannot be reviewed until you fix the following:• Your uploaded proof has already been used previously." I am wondering what should I do next to fix this problem. Thanks so much.

**Comment by vacisdev** [comment]
Check errors in console, clear browser cache and cookies, try different browser, contact github support if nothing helped.

**Comment by samus-aran** [comment]
Hi @sw3869 👋 

Check out our updated [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352)🎒. These links should help you with your scenario 👍 

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)
